assets/dev/scss/frontend/_container.scss

```diff
@@ -1,4 +1,18 @@
 .e-con {
+	// Todo: remove in future version.
```

Comment:
do you know when? maybe add a more precise condition ?

Reply:
@TzviRabinovitch See comment lower in this file.